src/transactions/transaction.rs

```diff
@@ -94,20 +94,21 @@ pub fn trace_to_transaction(trace: &TransactionTrace) -> Result<Transaction, Tra
                 input,
             })
         }
+        TxType::Eip4844 => unimplemented!(),
```

Comment:
@severiano-sisneros , @pedrohba1 , this was one moment I wanted to point out to you both. Do we want this to panic if we bit this variant or simply do nothing?

Reply:
The thing with EIP4844 is that they were not present pre-merge. It is the blob transaction type I think. We can deal with it when we start validating post merge.

Reply:
Thanks, I've created an [issue to implement EIP4844](https://linear.app/semiotic/issue/TRU-203/implement-eip-4844-in-transactionstransactionrs).